src/js/Content/Features/Community/Games/FCommonGames.js

```diff
@@ -32,7 +32,7 @@ export default class FCommonGames extends Feature {
     }
 
     async _loadCommonGames() {
-        if (this._commonGames !== null) { return; }
+        if (!this._commonGames) { return; }
```

Comment:
It's supposed to check for `undefined` but I changed it to a loose falsy check because it's done similarly elsewhere in the extension